src/main/java/Duke.java

```diff
@@ -6,5 +9,101 @@ public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
+            String firstCommandType = null;
+            String secondCommandType = null;
+            String inputLine = in.nextLine();
+
+            String[] commandType = inputLine.split(" ", 2);
+            if (commandType.length > 1) {
+                firstCommandType = commandType[0];
+                secondCommandType = commandType[1];
+            }
+            else {
+                firstCommandType = commandType[0];
+                secondCommandType = "";
+            }
+
+            switch (firstCommandType) {
+            case COMMAND_LIST_WORD:
+                System.out.println(divider);
+                System.out.println(MESSAGE_LIST_TASKS);
+                for (int i = 0; i < Tasks.getTotalTask(); ++i) {
+                    System.out.print("  " + listOfTasks[i].getTaskNumber()+ ". " + listOfTasks[i] + "\n");
+                }
+                System.out.println(divider);
+                break;
+            case COMMAND_BYE_WORD:
+                System.out.println(divider);
+                System.out.println(MESSAGE_GOODBYE);
+                System.out.println(divider);
+                break;
+            case COMMAND_DONE_WORD:
+                int ID = Integer.parseInt(secondCommandType) - 1;
+                listOfTasks[ID].markAsDone(true);
+                System.out.println(divider);
+                System.out.println(MESSAGE_MARK_DONE);
+                System.out.println("\t\t" + listOfTasks[ID].toString());
+                System.out.println(divider);
+                break;
+            case COMMAND_EVENT_WORD:
+                String[] events = secondCommandType.split("/at");
+                Event newEvent = new Event(events[0], events[1]);
+                System.out.println(divider);
+                System.out.println(MESSAGE_ADD_TASK+ "\n" + "  " + newEvent);
+                printTaskCount();
+                System.out.println(divider);
+                listOfTasks[Tasks.getTotalTask() - 1] = newEvent;
+                break;
+            case COMMAND_DEADLINE_WORD:
+                String[] deadlines = secondCommandType.split("/by");
+                Deadline newDeadLine = new Deadline(deadlines[0], deadlines[1]);
+                System.out.println(divider);
+                System.out.println(MESSAGE_ADD_TASK+ "\n" + "  " + newDeadLine);
+                printTaskCount();
+                System.out.println(divider);
+                listOfTasks[Tasks.getTotalTask()] = newDeadLine;
+                break;
+            case COMMAND_TODO_WORD:
+                description = secondCommandType;
+                Todo newToDo = new Todo(description, "T");
+                System.out.println(divider);
+                System.out.println(MESSAGE_ADD_TASK+ "\n" + "  " + newToDo);
+                printTaskCount();
+                System.out.println(divider);
+                listOfTasks[Tasks.getTotalTask() - 1] = newToDo;
```

Comment:
These above lines could be refactored into one function because it is a code quality violation